behave/features/steps/foxadmin.py

```diff
@@ -186,5 +187,57 @@ def step_impl(context):
         f"User should not be superuser"
 
 
+@step('I select the newly created user from the list')
+def step_impl(context):
+    # Click on newly created operator
+    context.execute_steps(f'''
+    Given I select the link "{USERS['FOX_ADMIN_NEW_USER']['username']}"
+    ''')
+
+
+@step('I select \'Delete\' in the user\'s details page')
+def step_impl(context):
+    btn = context.helperfunc.find_by_xpath(f"//a[text()='Delete']")
+    assert btn is not None
+    btn.click()
+
+
+@step(u'I am taken to the user\'s details page')
+def step_impl(context):
+    user_input = context.helperfunc.driver().find_element_by_xpath("//*[@id='id_username']")
+    assert user_input.get_attribute('value') == USERS['FOX_ADMIN_NEW_USER']['username']
+    assert_header_on_page("Change user", context)
+
+
+@step(u'I am taken to the \'Are you sure page\'')
+def step_impl(context):
+    # Cannot rely on checking page URL because PK could be different on each test run.
+    header = context.helperfunc.driver().find_element_by_xpath("//*[@id='content']/h1[text()='Are you sure?']")
+    assert header is not None
+    user_name = context.helperfunc.driver().find_element_by_xpath(f"//ul/li[text()='User: ']/a[text()='"
+                                                                  f"{USERS['FOX_ADMIN_NEW_USER']['username']}']")
+    assert user_name is not None
+
+
+@step(u'I confirm the user has been deleted from the list of users')
+def step_impl(context):
+    xpath = f"//a[text()='{USERS['FOX_ADMIN_NEW_USER']['username']}']"
+    try:
+        context.helperfunc.driver().find_element_by_xpath(xpath)
+    except NoSuchElementException as ex:
+        pass
+
+
+@step(u'I select the \'Yes, I\'m sure\'')
+def step_impl(context):
+    yes_btn = "//input[@type='submit']"
+    context.helperfunc.click_button(By.XPATH, yes_btn)
+
+
+@step(u'I am on the call centre dashboard')
```

Comment:
There is already same/similar step defined here https://github.com/ministryofjustice/cla-end-to-end-tests/blob/main/behave/features/steps/test_steps.py#L8

Reply:
Good thing there is already step, I just couldn't find that when I was looking.